experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/LivePersonAgentBot.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Duplicate from the other file; same comment applies. Let's get rid of .Net core 2.1 for new stuff; new stuff needs to be .Net Core 3.1.